vnext/docs/ParityStatus.md

```diff
@@ -0,0 +1,38 @@
+# Component & Module API Parity Status
+This doc reports the status progress of core components modules/APIs integration for vnext. The components and modules tracked here will be in alignment with the [React Native Lean Core](https://github.com/facebook/react-native/issues/23313) effort.
+
+Ones in **bold** are in the milestone we're currently targetting.
+
```

Comment:
May be good to add a note for folks to submit issues if they see missing features.

Reply:
I added as a call out! Also linked to the nvext issue submission page for ease of issue creation :)